avm

```diff
@@ -18,16 +18,10 @@ if [ -z "$1" ]; then
     exit 1
 fi
 
-# Mount .azure directory if it exists
-AZURE_VOLUME=""
-if [ -d "$HOME/.azure" ]; then
-  AZURE_VOLUME="-v $HOME/.azure:/home/runtimeuser/.azure"
-fi
-
 # Check if we are running in a container
 # If we are then just run make directly
 if [ -z "$AVM_IN_CONTAINER" ]; then
-  $CONTAINER_RUNTIME run --pull always --user "$(id -u):$(id -g)" --rm $AZURE_VOLUME -v "$(pwd)":/src -w /src -e GITHUB_TOKEN -e GITHUB_REPOSITORY -e ARM_SUBSCRIPTION_ID -e GITHUB_REPOSITORY_OWNER mcr.microsoft.com/azterraform make "$1"
```

Comment:
is there a standard for this? most of the other AVMs I have looked at seem to match the previous notation